### PR TITLE
a proper fix for anyrubber product outputting silicon rubber on macerate

### PR DESCRIPTION
**Pull request by Glease**
Fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11134

### CODE REVIEW

File: src/main/java/gregtech/api/util/GT_UtilityClient.java [hunk 1]
```diff
@@ -43,4 +61,105 @@ public static List<String> getTooltip(ItemStack aStack, boolean aGuiStyle) {
 		}
 	}
 
+    public static void debugPrintGLEnable() {
```
Comment (miozune) [inline src/main/java/gregtech/api/util/GT_UtilityClient.java:64]: looks totally unrelated

File: src/main/java/gregtech/api/enums/Materials.java [hunk 2]
```diff
@@ -1362,6 +1362,7 @@ private static void setMaceratingInto() {
         Peanutwood.setMaceratingInto(Wood);
         WoodSealed.setMaceratingInto(Wood);
         NetherBrick.setMaceratingInto(Netherrack);
+        AnyRubber.setMaceratingInto(Rubber);
```
Comment (miozune) [inline src/main/java/gregtech/api/enums/Materials.java:1365]: and `AnySyntheticRubber` too

File: src/main/java/gregtech/api/util/GT_UtilityClient.java (hunk 1 above)
Comment (Glease) [reply src/main/java/gregtech/api/util/GT_UtilityClient.java:64]: yeah it probably shouldn't be there..

File: src/main/java/gregtech/api/enums/Materials.java (hunk 2 above)
Comment (Glease) [reply src/main/java/gregtech/api/enums/Materials.java:1365]: Pretty sure normal rubber is not synthetic, and silicon rubber and StyreneButadieneRubber doesn't seem to have a clear difference recipe wise.

**Review by Dream-Master — APPROVED** [review]
